packages/core/mesh/network-manager/src/signal/swarm-messenger.ts

```diff
@@ -155,15 +155,15 @@ export class SwarmMessenger implements SignalMessenger {
     };
     const answer = await this._onOffer(offerMessage);
     answer.offerMessageId = message.messageId;
-    await this._sendReliableMessage({
+    return this._sendReliableMessage({
```

Comment:
Again, why return?

Reply:
It will log an error on one end https://github.dev/dxos/dxos/blob/a2d2a302ecc5ce39b0302926741a5886fdd7497e/packages/core/mesh/network-manager/src/swarm/swarm.ts#L103-L104

And on another end, the offer will fail with timeout and the connection will be closed 
https://github.dev/dxos/dxos/blob/mykola/drop-connection-if-signal-fails/packages/core/mesh/network-manager/src/swarm/peer.ts#L172-L174